### DISCUSSION TITLE
Connect to new WiFi

### DISCUSSION THREAD

**Discussion by MEW007** [arnemauer/Ducobox-ESPEasy-Plugin#153] [Q&A]
I installed a new WiFi network with a different SSID and password. How can I reconfiger the Serial Gateway so it can be connected to the new WiFi?

**Comment by Maarten71** [comment]
Still an issue or did you manage?

**Reply by MEW007** [reply]
Yes, still not figured it out.

**Comment by Maarten71** [comment]
I guess you know the ip address on the current network?

**Comment by arnemauer** [comment]
If espeasy can't connect to the wifi network it will start an wifi accesspoint. You can connect with that wifi network and change the wifi settings. You can follow this steps: https://github.com/arnemauer/Ducobox-ESPEasy-Plugin/wiki/Configureren---Wifi-instellen
